vmdb/lib/workers/replication_worker.rb

```diff
@@ -114,6 +75,17 @@ def check_replicate_process
     end
   end
 
+  def start_uninstall
+    start_rubyrep(:uninstall)
+  end
+
+  def start_rubyrep(verb)
+    raise "Cannot call start_rubyrep if a process already exists" if rubyrep_alive?
+    $log.info("#{self.log_prefix} Starting #{verb.to_s.humanize} Process")
+    start_rubyrep_process(verb)
+    $log.info("#{self.log_prefix} Started  #{verb.to_s.humanize} Process")
+  end
+
   def start_rubyrep_process(verb)
```

Comment:
Nope.  I felt that raw start of the process was nice as standalone method.   I just changed the caller to call directly instead of dynamically